test/utils/GeneratorAPI1832.scala

```diff
@@ -110,11 +508,113 @@ trait GeneratorAPI1832 extends Matchers with OptionValues with ResponseGenerator
   }
 
 
-  def generateGET1832ResponseAndUserAnswers: Gen[Tuple2[JsValue, EventType]] = {
+  def generateGET1832ResponseAndUserAnswers: Gen[(JsValue, EventType)] = {
     for {
       chosenEventType <- Gen.oneOf[EventType](Seq(EventType.Event22, EventType.Event23))
     } yield {
       Tuple2(generatedPayload(chosenEventType), chosenEventType)
     }
   }
 }
+
+
+//noinspection ScalaStyle
```

Comment:
Why we switching off scalastyle here?

Reply:
Addressed here: https://github.com/hmrc/pension-scheme-event-reporting/pull/74/commits/7192c98d8dcdc93ebebf3355c39e0b712a2a81f6